app/controllers/admin/merchants_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::MerchantsController < ApplicationController
   def index
     @enabled_merchants = Merchant.filter_merchant_status(1)
```

Comment:
I know this wasn't your change Joel, but just noticed the two instance variables above.  I think we should see if we can just call those filter_merchant_status(x) in the view rather than the controller.

Reply:
I bet we can condense that to just @merchant and then call the merchant filter method from inside the view perhaps?

Reply:
Yeah I think this one is mine. I left a comment on my PR that I was hoping y'all could give some input on the pros/cons of both ways because I wasn't sure which way was better 😅

Reply:
I'm sorry I missed that Parker!  We can go back to that and try it.